external/llvm-project/mlir/lib/Dialect/GPU/IR/GPUDialect.cpp

```diff
@@ -1395,14 +1383,6 @@ void GPUFuncOp::setWorkgroupAttributionAttr(unsigned index, StringAttr name,
                      getWorkgroupAttribAttrsAttrName());
 }
 
-void GPUFuncOp::setPrivateAttributionAttr(unsigned index, StringAttr name,
```

Comment:
Why's this removed? I could've sworn it exists upstream too

Reply:
It does, it is called `setPrivateAttirbutionAttrs` (with an `s` at the end) :)

Reply:
... right, that was me upstreaming this